draft/context.jsonld

```diff
@@ -1,7 +1,51 @@
 {
   "@context": {
     "@version": 1.1,
+    "ocfl": "https://ocfl.io/draft/spec#",
+    "dcterms": "http://purl.org/dc/terms/",
     "id": "@id",
-    "type": "@type"
+    "type": "@type",
+
+    "Object": "ocfl:Object",
+    "Version": "ocfl:Version",
+
+    "digestAlgorithm": {
+      "@id": "ocfl:digestAlgorithm"
+    },
+    "head": {
+      "@id": "ocfl:head"
+    },
+
+    "manifest": {
+      "@id": "ocfl:manifest"
+    },
+    "versions": {
+      "@id": "ocfl:versions",
+      "@container": "@list",
+      "@type": "ocfl:Version"
+    },
+    "fixity": {
+      "@id": "ocfl:fixity"
+    },
+
+    "version": {
+      "@id": "ocfl:version"
+    },
+    "created": {
+      "@id": "dcterms:created",
+      "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+    },
+    "message": {
+      "@id": "ocfl:message"
+    },
+    "user": {
+      "@id": "ocfl:user"
+    },
+    "name": {
+      "@id": "ocfl:name"
+    },
+    "address": {
+      "@id": "ocfl:address"
```

Comment:
`address`: are we leaving this open to any kind of address? email, mailing, etc?

Reply:
I think it's safe to restrict it to electronic addresses, but whether that includes e-mail, twitter handles, etc. should be discussed.

Reply:
Yeah, am following the spec here which seems OK for now. I think we should think a bit harder about the metadata elements after we have this draft done, they seem rather arbitrary to me at the moment and I'm really not sure that they should all be MUST -- it would be perfectly reasonable to decide to store such metadata within the versions according to local convention, it is not necessary for handling the objects per se